src/VCR/LibraryHooks/CurlHook.php

```diff
@@ -203,7 +218,7 @@ public static function curlExec($curlHandle)
      * @param resource $multiHandle A cURL multi handle returned by curl_multi_init().
      * @param resource $curlHandle  A cURL handle returned by curl_init().
      *
-     * @return integer Returns 0 on success, or one of the CURLM_XXX errors code.
+     * @return void
```

Comment:
Why changing this to `void` and not keep the original return type?
http://php.net/manual/en/function.curl-multi-add-handle.php